rollup.config.js

```diff
@@ -24,7 +24,7 @@ const postcssReporter = require('postcss-reporter');
 const { pkg } = readPkgUp.sync({
   cwd: fs.realpathSync(process.cwd()),
 });
-
+const importReplacements = [{ original: 'lodash', replacement: 'lodash-es' }];
```

Comment:
Is this something we want todo in the app too?

Reply:
I don’t see why not 😉